keyboards/preonic/keymaps/boy314/keymap.c

```diff
@@ -21,17 +21,17 @@
 // define layers
 #define _QWERTY 0
 #define _DVORAK 1
-#define _OSU 2
+#define _ARROWS 2
 #define _LOWER 10
 #define _RAISE 11
 #define _ADJUST 12
 
 // macro shortcuts
 #define QWERTY TO(_QWERTY)
 #define DVORAK TO(_DVORAK)
-#define OSU TO(_OSU)
 #define LOWER M(_LOWER)
 #define RAISE M(_RAISE)
+#define ARROWS TO(_ARROWS)
 
 #define _______ KC_TRNS
```

Comment:
These two keycodes are already defined in QMK, there's no need to have them here.